draft-ietf-ohai-ohttp.md

```diff
@@ -249,7 +249,8 @@ Oblivious HTTP is more costly than a direct connection to a server.  Some costs,
 like those involved with connection setup, can be amortized, but there are
 several ways in which Oblivious HTTP is more expensive than a direct request:
 
-* Each request requires at least two regular HTTP requests, which adds latency.
+* Each request requires at least two regular HTTP requests, which adds
+  processing latency.
```

Comment:
It could be latency do the different path itself, too, so I would keep this as-is. But I don't feel strongly.

Reply:
I'm happy with a different wording here, but even latency due to a different path is quite different from having extra round trips. I'd like to make sure people don't think that it always has extra latency. Honestly, if your relay and gateway have a more optimized path, you may have *lower* latency.

Reply:
Yes I agree with this point it _could_ be faster

Reply:
I agree. The intro to this list says that OHTTP _adds_ cost, but this isn't true. It's a different cost, and that cost may be less than that of a direct connection. Maybe we can change the intro to say that OHTTP has different performance characteristics than a direct connection which vary based on things like having primed connections, more optimized paths, etc?